messaging/react-typescript-webpack/README.md

```diff
@@ -0,0 +1,33 @@
+# FCM JS SDK React + TypeScript + Webpack Example
+
+The Firebase Cloud Messaging quickstart demonstrates how to:
+
+- Request permission to send app notifications to the user.
+- Receive FCM messages using the Firebase Cloud Messaging JavaScript SDK.
+- Use React and TypeScript with Firebase Cloud Messaging.
+- Use Webpack and TypeScript to bundle your window and service worker code while keeping type safety.
+
+Read more about Firebase Cloud Messaging [here](https://firebase.google.com/docs/cloud-messaging/).
+
+## Getting Started
+
+1. Set up your Firebase Project.
+   1. Create your project on the [Firebase Console](https://console.firebase.google.com).
```

Comment:
If it helps, you can link straight to the page by putting a _ instead of the project id. The user is prompted to pick a project then taken to the right page.

So here https://console.firebase.google.com/project/_/settings/cloudmessaging/ would work?